lnwallet/channel.go

```diff
@@ -2127,6 +2155,38 @@ func NewBreachRetribution(chanState *channeldb.OpenChannel, stateNum uint64,
 				return nil, err
 			}
 		}
+
+		// At this point, we'll check to see if we need any extra
+		// resolution data for this output.
+		resolveReq := ResolutionReq{
+			ChanPoint:      chanState.FundingOutpoint,
+			ShortChanID:    chanState.ShortChanID(),
+			Initiator:      chanState.IsInitiator,
+			FundingBlob:    chanState.CustomBlob,
+			Type:           input.TaprootRemoteCommitSpend,
+			CloseType:      Breach,
+			CommitTx:       spendTx,
+			SignDesc:       *br.LocalOutputSignDesc,
+			KeyRing:        keyRing,
+			CsvDelay:       ourDelay,
```

Comment:
I think by the description of `BreachCsvDelay`: 
```
// BreachCsvDelay is the CSV delay for the remote output. This is only
// set when the CloseType is Breach. This indicates the CSV delay to
// use for the remote party's to_local delayed output, that is now
// rightfully ours in a breach situation.
```

it is correct as-is

Reply:
I also think this is correct because I remember Laolu fixing this with a follow-up commit at some point, so I think we have an itest for it. Can you please confirm @Roasbeef?

Reply:
So we need two CSV delays here:
  * One which in most cases is just 1, due the CPFP carve out. 
  * When handling breaches, we need to reconstruct the script of the remote party, which will use the CSV of their actual commitment transaction.